change-address/LETTERSPECS/BANNO.CHANGE.ADDR.V1.CFG

```diff
@@ -128,42 +128,62 @@ MT: 0
 *  DEFAULT: blank (Test mode deactivated)
 TA:
 *
-* SEND EMAIL 
+*  OVERRIDE CONVERSATIONS
+*  If set to true and any error occurs during an address change, the
+*  address information and error are written to a note record on the
+*  account and an email is sent to the address specified in the 
+*  CONV OVERRIDE EMAIL parameter below.
+*  VALID VALUES: YES or NO
+*  DEFAULT: NO (or left blank) 
+*
+OC:
+*
+*  CONVERSATIONS OVERRIDE EMAIL
+*  If OVERRIDE CONVERSATIONS above is set to true, a valid email
```

Comment:
Technically it is being validated but because the email is their only notification it was invalid and they'll never get it, I added a note that indicates that with !!!'s around it.